### PR TITLE
Fix MultipleLines example

### LINKED ISSUE #4388: Correct the plot angles to show that there are multiple lines
### Describe the feature you would like to be added.

The MultipleLines docstring's example does not show that the output image is poorly oriented and has multiple lines.
image

### Links to VTK Documentation, Examples, or Class Definitions.

https://docs.pyvista.org/version/dev/api/utilities/_autosummary/pyvista.MultipleLines.html

### Pseudocode or Screenshots

Please correct the plot angles to show that there are multiple lines.

### PR DESCRIPTION
**Pull request by AndCotOli**
### Overview

Fixes  #4388 by using the plotter to rotate the camera.

### Details
Rotates the camera by 45 degrees and zooms out to see both lines.

### CODE REVIEW

**Comment by tkoyama010** [comment]
@AndCotOli
Thank you for your first contribution! Could you help us by following the steps in this section to make style corrections?
https://docs.pyvista.org/version/dev/extras/developer_notes.html#style-checking

**Comment by banesullivan** [comment]
We could also use the pre-commit bot to fix the PR automatically

File: pyvista/core/utilities/geometric_objects.py [hunk 1]
```diff
@@ -507,7 +507,11 @@ def MultipleLines(points=[[-0.5, 0.0, 0.0], [0.5, 0.0, 0.0]]):
     >>> mesh = pyvista.MultipleLines(
     ...     points=[[0, 0, 0], [1, 1, 1], [0, 0, 1]]
     ... )
-    >>> mesh.plot(color='k', line_width=10)
+    >>> plotter = pyvista.Plotter()
+    >>> actor = plotter.add_mesh(mesh)
```
Comment (tkoyama010) [inline pyvista/core/utilities/geometric_objects.py:511]:
```suggestion
    >>> actor = plotter.add_mesh(mesh, color='k', line_width=10)
```
I added options to match the appearance of the mesh with that before the change. Can you confirm that this is not a problem and approve the change?

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

**Review by tkoyama010 — APPROVED** [review]
LGTM. Great PR!

**Pull request merged**